AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -1143,57 +1136,32 @@ - (void)nodeDidRelayout:(ASCellNode *)node sizeChanged:(BOOL)sizeChanged
     return;
   }
   
-  BOOL queued = (_queuedNodeSizeInvalidationContext != nil);
-  if (!queued) {
-    _queuedNodeSizeInvalidationContext = [[_ASCollectionViewNodeSizeInvalidationContext alloc] init];
-    
-    __weak __typeof__(self) weakSelf = self;
-    dispatch_async(dispatch_get_main_queue(), ^{
-      __typeof__(self) strongSelf = weakSelf;
-      if (strongSelf) {
-        [strongSelf requeryNodeSizes];
-      }
-    });
+  NSIndexPath *indexPath = [self indexPathForNode:node];
```

Comment:
Hmm it would be nice but a major pain and very expensive to update the index paths of all nodes whenever an update is performed. We could maybe benefit from using NSOrderedSet but probably not. According to https://github.com/Necropolis/Set/blob/master/NSSet-PerformanceDemonstration/NSSet-PerformanceDemonstration.md NSOrderedSet.indexOfObject is about 25x faster on indexOfObject: but about 9x slower on addObject. As always Apple is silent about the theoretical performance characteristics.